docs/content/dagster-cloud/account/setting-up-alerts.mdx

```diff
@@ -12,9 +12,13 @@ In this guide, we'll walk you through configuring job alerts in Dagster Cloud.
 
 ## Understanding alert policies
 
-Alert policies define which jobs will trigger an alert, the conditions under which an alert will be sent, and how the alert will be sent.
+Alert policies define which events will trigger an alert, the conditions under which an alert will be sent, and how the alert will be sent.
 
-An alert policy includes a set of configured tags. If an alert policy has no configured tags, all jobs will be eligible for that alert. Otherwise, only jobs that contain all the tags for a given alert policy are eligible for that alert.
+Currently, alerts can be triggered by job failure/success, or schedule/sensor tick failure.
+
+Job-based alert policies includes a set of configured tags. If an alert policy has no configured tags, all jobs will be eligible for that alert. Otherwise, only jobs that contain all the tags for a given alert policy are eligible for that alert.
+
+Alert policies created for schedule/sensor tick failure will apply to all schedules / sensors. However, you will only receive alerts when the schedule/sensor changes from a state of succeeding to failing, so subsequent failures will not trigger new alerts.
```

Comment:
```suggestion
Currently, alerts can be triggered by job failure/success, or schedule/sensor tick failure.

- **Job-based alert policies** include a set of configured tags. If an alert policy has no configured tags, all jobs will be eligible for that alert. Otherwise, only jobs that contain all the tags for a given alert policy are eligible for that alert.

- **Alert policies created for schedule/sensor tick failure will apply to all schedules/sensors**. However, you will only receive alerts when the schedule/sensor changes from a state of succeeding to failing, so subsequent failures will not trigger new alerts.
```

Fixes a typo and puts these items into a bulleted list for readability